etl/steps/data/garden/regions/2023-01-01/regions.py

```diff
@@ -0,0 +1,149 @@
+"""Region definitions, aliases, members, historical transitions and codes.
+
+This step will load the adjacent files of region definitions and codes, and create a garden dataset with tables:
+* "definitions": Region definitions (name, country type, etc.).
+* "aliases": Region aliases (i.e. variants of the region name).
+* "members": Region members (roughly, sub-regions that would need to be added up when aggregating data for the region).
+* "related": Other possible region members to be aware of (e.g. regions with an unclear official status that may be
+  members of the region according to some data providers).
+* "transitions": Historical transitions between regions.
+* "legacy_codes": Legacy codes (e.g. wikidata code).
```

Comment:
I'd keep them as it might be useful in the future for updating old datasets or redoing datasets that come from literature. Wikidata codes might be useful at some point to fetch data from there (e.g. capitals for countries)